autocertkit/operations_tests.py

```diff
@@ -233,42 +233,37 @@ def test_crashdump(self, session):
         """Check crashdump is created properly."""
         log.debug("Running Crashdump test.")
         if not get_reboot_flag():
-            numberofcds = len(retrieve_crashdumps(session))
             tc_info = {}
             tc_info['device'] = self.config['device_config']['udid']
             tc_info['test_class'] = self.__class__.__module__ + '.' + self.__class__.__name__
             tc_info['test_method'] = 'test_crashdump'
-            tc_info['cds'] = numberofcds
+            tc_info['existing_crashdumps'] = [cd['timestamp'].strftime("%Y-%m-%d %H:%M:%S") for cd in retrieve_crashdumps(session)]
```

Comment:
Right, test is checking for 'new' crashdump after crash-reboot and will only fail if it doesn't see a 'new' crashdump. Additionally it will check timestamp for log and duration of crash, which is logged but wont fail if they dont. a warning will be logged though.